test/specs/commonTests/implementsWrapperProp.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import { mount, ReactWrapper } from 'enzyme'
+
+import Slot from 'src/components/Slot'
+import Segment from 'src/components/Segment'
+import { ShorthandValue } from 'utils'
+
+export interface ImplementsWrapperPropOptions {
+  wrapppedComponentSelector: any
+  wrappperComponentSelector?: any
+}
+
+const implementsWrapperProp = <P extends { wrapper: ShorthandValue }>(
+  Component: React.ReactType<P>,
+  options: ImplementsWrapperPropOptions,
+) => {
+  const { wrapppedComponentSelector, wrappperComponentSelector = Slot.defaultProps.as } = options
+
+  const wrapperTests = (wrapper: ReactWrapper) => {
+    expect(wrapper.length).toBeGreaterThan(0)
+    expect(wrapper.find(wrapppedComponentSelector).length).toBeGreaterThan(0)
+  }
+
+  describe('"wrapper" prop', () => {
+    it('does not wrap the component when set to false', () => {
```

Comment:
we should eliminate this test, as we've agreed to not support this scenario for now (`element should always be wrapped`)

Reply:
agreed to remove the test

Reply:
done